frontend/containers/faq-page/faq-list/component.tsx

```diff
@@ -0,0 +1,82 @@
+import { FC } from 'react';
+
+import { FormattedMessage } from 'react-intl';
+
+import cx from 'classnames';
+
+import { FaqListProps, ListItemProps, ItemValidationTypes } from './types';
+
+export const FaqList: FC<FaqListProps> = ({ className, level = 'one', children }: FaqListProps) => {
+  return (
+    <ol
+      className={cx({
+        'ml-6': true,
+        'my-4 font-medium list-decimal': level === 'one',
+        'font-normal list-lower-latin': level === 'two',
+        'text-sm list-lower-roman': level === 'three',
+        [className]: !!className,
+      })}
+    >
+      {children}
+    </ol>
+  );
+};
+
+export const ListItem: FC<ListItemProps> = ({
+  className,
+  title,
+  description,
+  inlineDescription = false,
+  mandatory = false,
+  validationType,
+  children,
+}: ListItemProps) => {
+  return (
+    <li
+      className={cx({
+        [className]: !!className,
+      })}
+    >
+      {title}
+      {inlineDescription && ': '}
+      {validationType && (
+        <>
+          {' '}
+          <span className="italic font-normal">
+            (
+            {validationType === ItemValidationTypes.Max600Chars && (
+              <FormattedMessage defaultMessage="max 600 characters" id="UhT/Dv" />
+            )}
+            {validationType === ItemValidationTypes.SelectMultiple && (
+              <FormattedMessage defaultMessage="select multiple" id="rsWAqe" />
+            )}
+            {validationType === ItemValidationTypes.SelectOne && (
+              <FormattedMessage defaultMessage="select one" id="gwLlk9" />
+            )}
+            {validationType === ItemValidationTypes.YesNo && (
+              <FormattedMessage defaultMessage="Yes / No" id="aNY6/x" />
+            )}
+            {validationType === ItemValidationTypes.Max36Months && (
+              <FormattedMessage defaultMessage="max 36 months" id="f9DtLB" />
+            )}
+            )
+          </span>
+        </>
+      )}
+      {mandatory && ' *'}
+      {description && (
+        <p
+          className={cx({
+            'font-light text-gray-600': true,
```

Comment:
```suggestion
           'font-normal text-gray-700': true,
```

Reply:
The colors in figma differ a little from the ones we have in tailwind.config. The text should be rgba(117, 117, 117, 1), that corresponds to our text-gray-700, and the font weight should be 400, so font-normal